autoprotocol/constants.py

```diff
@@ -35,3 +35,9 @@
     (0.6243159222654036, -0.14191423819668741),
     (0.563207308766221, 0.5152506858187523),
 ]
+
+MEASUREMENT_MODES = ["mass", "volume"]
+
+VOLUMETRIC_UNITS = ["liter", "milliliter", "microliter"]
+
+MASS_UNITS = ["gram", "milligram", "microgram"]
```

Comment:
there are more volumetric and mass units than these. is it possible not to hardcode here?

Reply:
@joshhacksthings I know there are more volumetric and mass units. As an implementer AP specification, I read http://autoprotocol.org/specification/#units and assumed that, in the scientific world only mentioned units makes sense and hence scoped down to this list.

As per the ASC https://github.com/strateos/autoprotocol_standard_changes/blob/master/Proposed/asc052.md, in order to populate `measurement_mode` attribute, I need to refer to these constants. If you can point me to some library which can return some indication of measurement made based on unit, I will get rid of these constants. Till then, we may have to retain(I will add other possible units for volume and mass). Let me know.